modules/ROOT/pages/design-named-examples.adoc

```diff
@@ -195,7 +195,7 @@ fullName:
 
 If you do not want to use more than one example and you do not want to name your single example, use the `example` facet instead of the `examples` facet.
 
-Here, the `example` facet includes `fragment1.raml`. Neither the facet nor the fragment names the example.
+Here, the `example` facet includes `fragment1.yaml`. Neither the facet nor the fragment names the example.
```

Comment:
To what does "Here," refer? Should we say "In this example," or "In this case," instead?